gr-utils/python/modtool/modtool_rename.py

```diff
@@ -194,3 +189,23 @@ def _run_file_replace(self, filename, old, new):
         open(filename, 'w').write(cfile)
         self.scm.mark_file_updated(filename)
         return nsubs
+
+
+### COMMAND LINE INTERFACE ###
+@click.command('rename', short_help=ModToolRename().description)
+@ModTool.common_params
+@ModTool.block_name
+@click.argument('new-name', metavar="NEW-BLOCK-NAME", nargs=1, required=False)
+def cli(**kwargs):
+    """
+    \b
```

Comment:
?

Reply:
@skoslowski: Actually, `\b` is used to prevent rewrapping in a click command.
Documentation Link: http://click.pocoo.org/5/documentation/#preventing-rewrapping